pkg/rpc/server.go

```diff
@@ -225,6 +227,38 @@ Methods:
 		} else {
 			results = "Invalid public account address"
 		}
+	case "getrawtransaction":
+		var err error
+
+		param0, exists := reqParams.ValueAtAndType(0, "string")
```

Comment:
can you replace 
`param0, exists := reqParams.ValueAtAndType(0, "string")`
to
`param0, exists := reqParams.ValueWithType(0, "string")`

Reply:
ok